### ISSUE TITLE
Unable to see my vore tenants after the last Starbound update.

### ISSUE THREAD

**Issue by LeSarthois** [Sheights/StarboundSimpleVoreMod#68]
So, after Strabound (seemed to?) receiev a small update, about last week, I have a bug with the Vore Mod; I can no longer come clsoe to my vore tenants.If I try to teleport to the colony I made, I sipmply get the teleportation amination looping endlessly. If I beam to the planet, I land on it, but as soon as I come close to the colony, the game take me back to the ship.

I tried to use xbawks characters mod as it was mentionned before but that doesn't solve anything, and upgrading to the 8.5 version of the SSVM didn't improved anything.
Note that vore items and the Vore Mall are accessible (goodness thanks, as I have Tartii in my ship... else, my save game would be unplayable).
So I assume that the issue is only with the vore tenants as everything else works fine.
I have no other mods than SSVM and Xbawks, but I had the issue before I installed Xbawk's mod.

[starbound.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/673101/starbound.txt)

I joined the last log of my game. Note that I sometime get a crash of Starbound at launch but this seems to be random, and I couldn't get the game to crash since the update to 8.5.

**Comment by AmoraSilver** [comment]
Hmm, don't know how to possibly recreate this, but I do have a possible idea.  I will need five files from you:  universe.locks, universe.dat, universe.lock, the world file named :-514728983:-628715633:-53296871 and the clientcontext file that is usually on the top of the universe folder.

Or just zip up your entire universe folder and send it over here.

**Comment by LeSarthois** [comment]
I zipped the whole universe file. Anyway I only have one character and one save on this installation so it shouldn't be too hard to find :) 

Here is a Dropbox link, the file is apparently too big for GitHub : https://www.dropbox.com/s/d1apngqkpxuxu6o/universe.zip?dl=0

**Comment by AmoraSilver** [comment]
What version of the vore mod did you use before updating to the latest version?

**Comment by LeSarthois** [comment]
I used versions 8.3 and 8.4. I got unable to access the planet when using the 8.4 version.

**Comment by AmoraSilver** [comment]
Okay, I'm going to download the 8.3 version and downgrade and see if I can get the planet to become explore-able.

This won't affect my game as I use the unstable Starbound for bug testing.

**Comment by AmoraSilver** [comment]
Huh, was not expecting the world would still crash....

Now I can say for certain it is not the mod causing the crash, it is the Starbound update from 1.1 to 1.2 .  There is nothing I can really do as I do not know how to bring the game back to 1.1, sorry.

**Comment by LeSarthois** [comment]
I was afraid it was that kind of problem.
Ah well I can still redo it I guess, It's not like my whole progress is lost.

**Comment by AmoraSilver** [comment]
Yeah, just about every time Starbound does a major update, I tend to delete my whole universe folder after I make sure all my characters are on their ships and have salvaged anything I want to keep.  This is just so I don't run into a planet that becomes bugged like this as it tends to happen.  

I now do this thanks to having so many issues with the Frackin' Universe mod where it was outright required for me to start completely over when the 1.1 update came around.

**Comment by LeSarthois** [comment]
The problem being that I would have liked to save the colony stuff I made, with a sewer like place, and other stuff.

I guess I'll keep checking after updates but I assume that everything is corrupted and lost on this planet..

**Comment by LeSarthois** [comment]
Well, I removed the Simple Vore mod, went to the planet - of course, the vore tenants and some items were gone, but everything else was there. I reinstalled it and replaced all my tenants and it seems to go fine now.

I guess it's a case closed, and solved without too much loss :p

**Comment by AmoraSilver** [comment]
Oh?  I guess I'll remember that in the future if that kind of problem ever pops agin.

**Comment by LeSarthois** [comment]
Well, it happened again. 
I got the error first, then the game played normally (I installed Frackin Universe in the meantime)

This time I might know better what cause issue.
I got gobbled by a pred that was in one of those digestive seat critters. Because the pred got digested it poofed away, and I was stuck in place, unable to move.

I beamed up to my ship, and then the issue happen again, I cannot approach the colony again.

I've seen this happening quite a few time - tenants stuck in one place, but apparently some combinations of this issue, or me escaping such situation by beaming can cause troubles.

[starbound.log.4.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675826/starbound.log.4.txt)
[starbound.log.5.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675828/starbound.log.5.txt)
[starbound.log.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675829/starbound.log.txt)
[starbound.log.1.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675827/starbound.log.1.txt)
[starbound.log.2.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675830/starbound.log.2.txt)
[starbound.log.3.txt](https://github.com/Sheights/StarboundSimpleVoreMod/files/675831/starbound.log.3.txt)

**Comment by AmoraSilver** [comment]
Yeah, normally when you become stuck like that, either by what you said or a multivore pred seemingly overeats and dies, the best option is to save and quit then start back up.  That removes the digested status and allows you to move around again.

I don't know what is causing the planet error or how to fix it, sorry.

**Comment by LeSarthois** [comment]
It's strange that this issue happens to me only. Isn't there a way to clean the status of tenants with admin comments? I do'nt remember if the first time, that crash happened when I got stuck in a pred or not. Which mean that this issue is going to happen again eventually.

**Comment by AmoraSilver** [comment]
I don't think it has been only you.  I've had planets become bugged before and I've seen a ton of threads on the Starbound site about planets becoming inaccessible.  Main problem is there is no decent world viewer/editor out there to help combat buggy planets.  I only ever have seen a browser based one and it did not work at all for me.

**Comment by LeSarthois** [comment]
I see. I'll remove the vore furniture, that will probably reduce the issues.
